package.json

```diff
@@ -50,11 +50,12 @@
     "jwt-decode": "^2.2.0",
     "nodemailer": "^6.4.16",
     "oauth": "^0.9.15",
+    "pkce-challenge": "^2.1.0",
```

Comment:
This can probably be removed and we can create our own, added to be able to get started a bit faster

Reply:
`openid-client` has this built-in, we can utilize it when/if we finish #1105